api/server/secrets.go

```diff
@@ -0,0 +1,148 @@
+package server
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/libopenstorage/openstorage/secrets"
+)
+
+const (
+	secretKeyOkMsg   = "Secret Key set successfully"
+	secretLoginOkMsg = "Secrets Login Succeeded"
+	secretLoginCheck = "Secrets Login Check Succeeded"
+)
+
+// TODO: Add swagger yaml
```

Comment:
You can probably start adding this soon.

Reply:
yes Luis, since PR this PR is getting bulky I was planning to add mock test cases and swagger yaml  in new PR once this get merged.